consumer/stats_refresh_consumer.py

```
@@ -91,35 +128,130 @@ def start(self) -> None:
+        주입된 redis_client 가 있고 실제 RedisQueueClient 인 경우 동일 config 로
+        재인스턴스화. mock/fake 주입 시엔 singleton 경로로 fallback — mock 이
+        config 인자를 요구하지 않아 ``TypeError`` 로 튀는 문제 방어.
+        """
+        injected = self._injected_redis_client
+        reconnected = False
+        if injected is not None and hasattr(injected, "config"):
+            try:
+                try:
+                    injected.close()
+                except Exception:
+                    pass
```

Ruff (0.15.10)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)